src/robot/output/output.py

```diff
@@ -31,10 +32,12 @@ def __init__(self, settings):
 
     def _register_loggers(self, listeners, debugfile):
         LOGGER.register_context_changing_logger(self._xmllogger)
-        for logger in (Listeners(listeners), LibraryListeners(),
-                       DebugFile(debugfile)):
+        loggers = (Listeners(listeners), LibraryListeners(),
+                    DebugFile(debugfile))
+        for logger in loggers:
```

Comment:
Why was `loggers` created? It seems to be used only in the for loop so it could still be inlined.